crates/cubecl-hip/src/runtime.rs

```diff
@@ -112,6 +115,14 @@ fn create_client<M: WmmaCompiler<HipDialect<M>>>(
     );
     let mut device_props = DeviceProperties::new(&[Feature::Plane], mem_properties, topology);
     register_supported_types(&mut device_props);
+    // Not sure if there's a good way to check for support on HIP
```

Comment:
ROCm is only supported by newer devices, so I expect the features to be available most of the time (but it's a big guess).